README.rst

```diff
@@ -71,13 +71,38 @@ long-term compatibility with projects conflicts partially with providing and usi
 the latest Python versions bring.
 
 Since we pour more `effort into maintaining and developing icalendar <https://github.com/collective/icalendar/discussions/360>`__,
-we split the project into two:
+this is an overview of the versions:
 
-- `Branch 4.x <https://github.com/collective/icalendar/tree/4.x>`__ with maximum compatibility to Python versions ``2.7`` and ``3.4+``, ``PyPy2`` and ``PyPy3``.
-- `Branch master <https://github.com/collective/icalendar/>`__ with the compatibility to Python versions ``3.7+`` and ``PyPy3``.
+Version 6
+~~~~~~~~~
 
-We expect the ``master`` branch with versions ``5+`` receive the latest updates and features,
-and the ``4.x`` branch the subset of security and bug fixes only.
+Version 6 of ``icalendar`` switches the timezone implementation to ``zoneinfo``.
+
+    >>> dt = icalendar.Calendar.example("timezoned").walk("VEVENT")[0]["DTSTART"].dt
+    >>> dt.tzinfo
+    ZoneInfo(key='Europe/Vienna')
+
+If you would like to continue to use ``pytz`` and receive the latest updates, you
+can switch back:
+
+    >>> icalendar.use_pytz()
+    >>> dt = icalendar.Calendar.example("timezoned").walk("VEVENT")[0]["DTSTART"].dt
+    >>> dt.tzinfo
+    <DstTzInfo 'Europe/Vienna' CET+1:00:00 STD>
+
+`Branch master <https://github.com/collective/icalendar/>`__ with the compatibility to Python versions ``3.7+`` and ``PyPy3``.
+We expect the ``master`` branch with versions ``6+`` to receive the latest updates and features.
+
+Version 5
+~~~~~~~~~
+
+Version 5 contains the ``pytz`` only
```

Comment:
Thanks, I addressed this in commit 4eb94f5b888d66f5d69790ce5dfc53b8fb461323. Since #513 can be fixed soon, the other versions might not need mentioning much: Nobody took time to contribute to version 4 really.